style.css

```diff
@@ -0,0 +1,239 @@
+    padding: 7px;
+        }
+.choose-us p{
+    text-align: center;
+        }
+            
+.choose-us h1{
+    text-align: center;
+    padding: 5px;
+    margin-bottom: 2px;
+}
+.first-paragraph{
+    text-align: center;
+    padding: 5px;
+    margin-bottom: 25px;
+    color: brown;
+    font-size: 19px;
+}
+.div1{
+    height:200px;
+    width:250px;
+    border: 1px solid rgb(112, 112, 110);
+    padding:5px;
+    font-size: 15px;display: inline-block;
+    margin-right: 90px;
+    margin-top: 15px;
+    margin-left: 75px;
+    border-radius: 25px;
+
+   
+}
+ 
+
+.div2{
+    height:200px;
+    width:250px;
+    border: 1px solid rgb(112, 112, 110);
+    padding:5px;display: inline-block;
+    font-size: 15px;
+    margin: 40px 40px;
+    margin-top: 15px;
+    border-radius: 25px;
+}    
+.div3{
+    height:200px;
+    width:250px;
+    border: 1px solid rgb(112, 112, 110);
+    padding:5px;display: inline-block;
+    font-size: 15px;
+    margin: 40px 90px;margin-top: 15px;
+    border-radius: 25px;
+}  
+ .div4{
+    height:200px;
+    width:250px;
+    border: 1px solid rgb(112, 112, 110);
+    padding:5px;
+    font-size: 15px;display: inline-block;
+    margin: 40px 40px;
+    margin-top: 15px;
+    border-radius: 25px;
+}
+
+.gif{
+    width:100%;
+    height:15%;
+    position: relative;
+    background: url("1684622191787.png");
+    background-attachment: fixed;
+    background-repeat: no-repeat;
+    background-size: cover;   
+}
+.fa-chess-king{
+    height: 40px;
+    margin-left: 100px;
+    margin-top: 20px;
+
+}
+.fa-truck-field{
+    height: 40px;
+    margin-left: 100px;
+    margin-top: 20px;
+    font-size: 5px;
+}
+.fa-bowl-food{
+    height: 40px;
+    margin-left: 100px;
+    margin-top: 20px;
+}
+.fa-people-group{
+    height: 40px;
+    margin-left: 100px;
+    margin-top: 20px;
+}
+.gifheader{
+    position: absolute;
+    top: 20px;
+    font-size: 40px;
+    left:470px;
+    color: white;
+         }
+footer{
+    background-color:  rgb(219, 219, 226);
+    width: 100%;
+    height: 400px;
+    margin-top: 70px;
+    border: none;
+    background-image: url("1684616245900.jpg");
+    background-repeat: no-repeat;
+    background-size: 350px;
+    background-position: right bottom;
+    padding:60px; 
+}
+input{
+    border: 1px solid brown;
+    width:450px;
+    height: 30px;
+    padding: 15px;
+    margin-left: 280px;
+    background-color: white;
+    margin-top: 3px;
+}
+button{
+    border: none;
+    width:100px;
+    height: 33px;;
+    background-color: brown;
+    color: white;
+}
+footer >p, h4{
+    font-size: 20px;
+    margin-top:5px;
+}
+.tableheader{
+    width:69%;
+    margin-top: 40px;
+    height: 3px;
+    background-color: black;
+}
+table{
+    margin:40px;
+ }
+td{
+     padding-left: 100px;
+     padding-bottom: 10px;
+}
+th{
+    padding-left: 100px;
+    padding-bottom: 10px;
+}
+.fa-utensils{
+    height: 20px;
+    background-color: yellow;
+    margin-right: 10px;
+
+}
+
+
+ 
```

Comment:
No media queries to ensure mobile responsiveness.

Reply:
Okay sir, I will work on it